src/post_install/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+if(APPLE AND WITH_APP_BUNDLE)
+    # Run macdeloyqt on the main app and any extra binaries and plugins as specified by the
+    # _macdeployqt_extra_binaries global property.
+    # All installl(TARGETS) calls should have already been called.
+    get_property(macdeployqt_extra_binaries GLOBAL PROPERTY _macdeployqt_extra_binaries)
+    kpxc_run_macdeployqt_at_install_time(
+        APP_NAME "${PROGNAME}"
+        EXTRA_BINARIES ${macdeployqt_extra_binaries}
+    )
+endif()
```

Comment:
Is there a reason why this needs its own subdirectory? I would prefer inlining this instead.

Reply:
From the commit message:

```
To ensure that macdeployqt is called after all the binaries are
installed, a new post_install subdirectory is needed to
circumvent CMake's limitation regarding the default order of
installation.
CMake first runs the current directory install() calls and then it's
child subdirectory ones. Because we want macdeployqt to be the last
install() call, it needs to be done inside a subdirectory that is
added last.
```

Reply:
Hmm. That's ugly. Is there no other way?

Reply:
Unfortunately not. 
The install order within a subdirectory is always:
1) run all install() commands of the current dir
2) run install commands of all child (and descendant) subdirectories.

With the current structure of the app being in src/CMakeLists.txt, the only way to ensure that macdeployqt is called after the app and all helpers/plugins are installed, is to put the install call in the last child subdirectory of src/CMakeLists.txt

Reply:
Ok, then leave it, but add a comment in the file why we need this directory.

Reply:
Couldn't you put it at the end of the base CMakeLists.txt since that would be executed last?

Reply:
Hmm. What's the minimum cmake version supported to build the project?
The main CMakeLists.txt claims 3.3.0 whereas the policy you mention was introduced in 3.14.
If the project supports CMake versions lower than 3.14, then the current approach is the fool-proof one.

Reply:
> Ok, then leave it, but add a comment in the file why we need this directory.

I added a comment in the latest commit.